docs/architecture/deployment/installation-guide.md

```diff
@@ -36,38 +36,105 @@ To install Prophecy, the following infrastructure requirements must be satisfied
+| **ATHENA**                                 |
+| `athena.adminPassword`\*                   | string                   | Password assigned to the Prophecy admin user                                                                                                                                                                                     | `admin@123`                         |
+| `athena.config`                            | -                        |                                                                                                                                                                                                                                  |                                     |
+| `athena.config.fromFile`                   | bool                     | Set this to `false` if you want to pass different environment and Prophecy application level configurations values inline here. If you keep it to `true`, the values are taken from the `../configs` directory of the helm chart | `true`                              |
+| `athena.config.athenaEnvs`                 | map[string]string        | Provide the values you wish to override in athena ENV variables. Passed in the format of `KEY: "value"`. Value is type sensitive. This field is only respected when fromFile is set to `false`.                                  | `./configs/athena-env-cluster.yaml` |
+| `athena.config.envs`                       | map[string]string        | Provide the values you wish to override in Prophecy application ENV variables. Passed in the format of `KEY: "value"`. Value is type sensitive. This field is only respected when fromFile is set to `false`.                    | `./configs/env-cluster.yaml`        |
+| `athena.config.prophecyCluster`            | prophecyCluster (custom) | Provide the values you wish to override in Prophecy cluster spec. This field is only respected when fromFile is set to `false`.                                                                                                  | `./configs/prophecy-cluster.yaml`   |
+| `athena.controlcenter`                     | -                        |                                                                                                                                                                                                                                  |                                     |
+| `athena.controlcenter.disabled`            | bool                     | Set to `true` if cluster is unable to connect to controlcenter.                                                                                                                                                                  | `false`                             |
```

Comment:
Description: Instead of unable to connect to controlcenter, you can mention "if you don't wish to connect to controlcenter for sharing monitoring statistics with Prophecy."

Reply:
```suggestion
| `athena.controlcenter.disabled`            | bool                     | if connectivity to controlcenter is not required                                                                                                                                                                  | `false`                             |
```